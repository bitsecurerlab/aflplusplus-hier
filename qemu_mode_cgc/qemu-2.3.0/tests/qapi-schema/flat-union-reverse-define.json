{ 'union': 'TestUnion',
  'base': 'TestBase',
  'discriminator': 'enum1',
  'data': { 'value1': 'TestTypeA',
            'value2': 'TestTypeB' } }

{ 'type': 'TestBase',
  'data': { 'enum1': 'TestEnum' } }

{ 'enum': 'TestEnum',
  'data': [ 'value1', 'value2' ] }

{ 'type': 'TestTypeA',
  'data': { 'string': 'str' } }

{ 'type': 'TestTypeB',
  'data': { 'integer': 'int' } }
