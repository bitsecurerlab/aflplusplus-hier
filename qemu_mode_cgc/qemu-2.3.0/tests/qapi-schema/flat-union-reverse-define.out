[OrderedDict([('union', 'TestUnion'), ('base', 'TestBase'), ('discriminator', 'enum1'), ('data', OrderedDict([('value1', 'TestTypeA'), ('value2', 'TestTypeB')]))]),
 OrderedDict([('type', 'TestBase'), ('data', OrderedDict([('enum1', 'TestEnum')]))]),
 OrderedDict([('enum', 'TestEnum'), ('data', ['value1', 'value2'])]),
 OrderedDict([('type', 'TestTypeA'), ('data', OrderedDict([('string', 'str')]))]),
 OrderedDict([('type', 'TestTypeB'), ('data', OrderedDict([('integer', 'int')]))])]
[{'enum_name': 'TestEnum', 'enum_values': ['value1', 'value2']}]
[OrderedDict([('type', 'TestBase'), ('data', OrderedDict([('enum1', 'TestEnum')]))]),
 OrderedDict([('type', 'TestTypeA'), ('data', OrderedDict([('string', 'str')]))]),
 OrderedDict([('type', 'TestTypeB'), ('data', OrderedDict([('integer', 'int')]))])]
